require.js

```diff
@@ -60,12 +65,6 @@
     aliases[to] = from;
   };
 
-  require.reset = function() {
```

Comment:
because we don't need it anymore. we don't want to re-initialize the modules that weren't changed
and these that did change, hmr-brunch makes sure they are reloaded — https://github.com/brunch/hmr-brunch/blob/master/runtime.js#L270